src/main/java/org/nuxeo/segment/io/SegmentIODataWrapper.java

```diff
@@ -60,7 +57,7 @@ public String getUserId() {
     }
 
     // code copied from com.github.segmentio.models.Props
```

Comment:
I don't know, and I don't know why we don't use the original method directly, I'll look at it